spec/clinic.v1.yaml

```diff
@@ -992,6 +992,42 @@ paths:
               $ref: '#/components/schemas/patient.v1'
       tags:
         - Clinics
+  /v1/clinics/{clinicId}/patient_counts/patient_tags:
```

Comment:
The authorization policy has those routes as `/v1/clinics/{clinicId}/counts/patient_tags` and `/v1/clinics/{clinicId}/counts/sites` in the comments - I don't know which one is correct, but both work.